WooCommerce/Classes/ViewRelated/Products/Edit Product/Cells/ImageAndTitleAndTextTableViewCell.swift

```diff
@@ -36,6 +37,7 @@ final class ImageAndTitleAndTextTableViewCell: UITableViewCell {
 
         init(title: String?,
              titleFontStyle: FontStyle = .body,
+             titleTintColor: UIColor? = nil,
```

Comment:
Nit: to avoid updating other cells, you might want to set the default  value to a common one (e.g. `.text`).

Reply:
There was just one point where `titleTintColor` was needed, and it was `DownloadableFileBottomSheetListSelectorCommon`. To match the behavior of `textTintColor` I preferred to set it to `nil` 👍